### PR TITLE
WIP: Time viewer tool bar with SVG icons

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR, following a discussion with @hoechenberger and strongly inspired by https://github.com/cbrnr/mnelab/pull/61, adds a tool bar with useful bindings to the features of `_TimeViewer`:

output

It's also an item of https://github.com/mne-tools/mne-python/issues/7162 since it adds buttons to the interface.

---

This is not compatible with https://github.com/mne-tools/mne-python/pull/7592

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
I considered removing the main menu but it's still useful when `time_viewer=False`

**Comment by GuillaumeFavelier** [comment]
I also thought about a button like the following:

image

that would display additionnal informations. It could be used to clarify the auto-scaling feature @larsoner (relevant for https://github.com/mne-tools/mne-python/pull/7247#issuecomment-583541940)

**Comment by larsoner** [comment]
@GuillaumeFavelier any way we can use unicode for the icons rather than bundling new binary images?

**Comment by hoechenberger** [comment]
> any way we can use unicode for the icons rather than bundling new binary images?

I don't think we'd find anything even closely as good-and-balanced-looking, tbh… also if you use a font, the icons could end up looking different on every system! -1 on unicode icons from my side for now :)

**Comment by hoechenberger** [comment]
@larsoner  What could work, potentially, are [FontAwesome](https://fontawesome.com) fonts, but personally I'm in favor of bundling SVG icons

**Comment by GuillaumeFavelier** [comment]
> any way we can use unicode for the icons rather than bundling new binary images?

It should be possible but it may have a totally different result on old version of PyQt5. It may require some particular processing like we did in https://github.com/mne-tools/mne-python/pull/7257#issuecomment-597711981

**Comment by hoechenberger** [comment]
@cbrnr Since you've been through this decision process personally with MNELAB, what's your stance on this?

**Comment by hoechenberger** [comment]
@GuillaumeFavelier Just tested on macOS and looks and works beautifully. I really like it! Also the availability of buttons now could compensate for some of the shortcomings of PyVista. For example, I don't think we'd need to hold on to the orientation slider, but add toolbar buttons for that (like in FreeView).

**Comment by GuillaumeFavelier** [comment]
> rather than bundling new binary images

Sorry, we can remove them from the `MANIFEST` since they're already built in `resources.py`

**Comment by cbrnr** [comment]
I'd go with SVG icons, I didn't even consider using unicode symbols. Just make sure to set `app.setAttribute(Qt.AA_UseHighDpiPixmaps)` (https://github.com/cbrnr/mnelab/pull/102/files).

**Comment by GuillaumeFavelier** [comment]
Just to be clear, to make it work with PyQt, only the built `resources.py` is necessary. So if the issue is about storing the images, they can be hosted in another repo. and reused only when the interface changes.

**Comment by GuillaumeFavelier** [comment]
Closing in favour of #7592 to move forward. At least a prototype is available here if needed.

File: MANIFEST.in [hunk 1]
```diff
@@ -58,6 +58,7 @@ exclude .github/ISSUE_TEMPLATE/feature_request.md
 exclude .github/PULL_REQUEST_TEMPLATE.md
 
 exclude mne.qrc
+recursive-exclude mne/images
 recursive-exclude mne/images *.svg
```
Comment (hoechenberger) [inline MANIFEST.in:62]: There seems to be some duplication here, right? I would suggest to use `prune mne/images`

Comment (GuillaumeFavelier) [reply MANIFEST.in:62]: Exactly what I wanted

**Comment by larsoner** [comment]
Works great, thanks @GuillaumeFavelier

**Pull request merged**